<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputView</class>
 <widget class="QWidget" name="InputView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btn1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="hideBtn">
     <property name="text">
      <string>hide</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backspaceBtn">
     <property name="text">
      <string>backspace</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
